docs/releasenotes/2.8.0.rst

```diff
@@ -0,0 +1,53 @@
+:orphan:
+
+Robocop 2.8.0
+================
+
+You can install the latest available version by running::
+
+    pip install --upgrade robotframework-robocop
+
+or to install exactly this version::
+
+    pip install robotframework-robocop==2.8.0
+
+
+Nested argument files (#770)
+---------------------------
+
+Nesting argument files was not allowed before. Now it is possible to import other argument file in the argument file::
+
+    --include some-rule
+    --argumentfile base_argument_file.txt
+
+Relative paths in the argument files (#693)
+-----------------------------------------------
+
+Argument files now support relative paths. The support was added for the ``pyproject.toml`` files before but it was
+missing from the argument files.
+
+It means that argument file can have paths relative to the argument file location::
+
+    content of config/argumentfile.txt:
+
+    --output robocop_output.txt # it will now refer to config/robocop_output.txt
+    --ext-rules robocop/rules.py # it will now refer to config/robocop/rules.py
+
+List only enabled or disabled rules #764
+------------------------------------------
+
+Robocop can list all rules with ``--list`` option (or ``--list-configurables`` to only list rules that can be
+configured). Our every growing list of rules made the output of this command hard to read. That's why it is now
+possible to filter rules with ``enabled`` or ``disabled`` special patterns::
+
+    robocop --list enabled
```

Comment:
Having the option accepting PATTERN and also some custom words like `enabled`/`disabled` may be confusing, because one may think that we're using these words as pattern, not as special controls.

I like the idea of being able to list only enabled/disabled rules, so I thought that maybe a good idea would be to make them uppercase, to indicate them as special specifiers? For example, we can allow for:
```
robocop --list ENABLED
```
which will print all enabled rules. But running:
```
robocop --list enabled
```
would treat it as a pattern. What do you think, @bhirsz?

I think it will make it more clear how to use this option properly and will not confuse the users.

Reply:
I agree, it will make better distinction. I'm not too fond of using the same way of filtering by pattern and enabled/disabled but that was the best thing I could think of without introducing too much complexity. In some way "enabled/disabled" is also pattern of sort so we can use this field as "filter by anything that matches criteria". I will make it uppercase